go.mod

```diff
@@ -1,169 +1,152 @@
 module github.com/dapr/dashboard
 
-go 1.17
+go 1.19
 
 require (
-	github.com/dapr/cli v1.7.1
-	github.com/dapr/dapr v1.7.0
+	github.com/dapr/cli v1.10.0-rc.3
+	github.com/dapr/dapr v1.10.0-rc.5
 	github.com/gorilla/mux v1.8.0
-	github.com/gorilla/websocket v1.4.2
-	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
+	github.com/gorilla/websocket v1.5.0
+	github.com/stretchr/testify v1.8.1
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
+	k8s.io/client-go v0.26.1
```

Comment:
Add a replace to use 0.25.2

Reply:
I cannot because `go mod tidy` reverts it back to the latest version.